test/suite/test_truncate01.py

```diff
@@ -36,35 +36,26 @@
 
 import wiredtiger, wttest
 from helper import confirm_empty
-from helper_disagg import DisaggConfigMixin, gen_disagg_storages
 from wtdataset import SimpleDataSet, ComplexDataSet, simple_key
 from wtscenario import make_scenarios
 
-class test_truncate_base(wttest.WiredTigerTestCase, DisaggConfigMixin):
-    conn_base_config = 'transaction_sync=(enabled,method=fsync),statistics=(all),statistics_log=(wait=1,json=true,on_close=true),' \
-                     + 'disaggregated=(page_log=palm),'
-    disagg_storages = gen_disagg_storages('test_truncate', disagg_only = True)
+class test_truncate_base(wttest.WiredTigerTestCase):
+    conn_base_config = 'transaction_sync=(enabled,method=fsync),statistics=(all),statistics_log=(wait=1,json=true,on_close=true),'
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.ignoreStdoutPattern('WT_VERB_RTS')
 
     def conn_config(self):
```

Comment:
Can we remove this function now?

Reply:
I tried, theres something finicky around using `conn_config` variable instead and updating it in the child classes.